site/content/en/guides/consumer/function/export/circleci/_index.md

```diff
@@ -37,7 +37,7 @@ Use `kpt pkg get` to fetch source files of this tutorial:
 git init
 git remote add origin https://github.com/<USER>/<REPO>.git
 # Fetch source files
-kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/function-export-blueprint example-package
```

Comment:
Need to actually rename the directory in order for this to work

Reply:
Done.